mappings/cBioPortal.yaml

```diff
@@ -0,0 +1,53 @@
+title: NF data dictionary to cBioPortal data dictionary for public clinical data
+description: This maps NF data elements at the variable level to cBioPortal data dictionary for the set of clinical elements that will be made public on the cBioPortal platform.
+source_scope: https://github.com/nf-osi/nf-metadata-dictionary/blob/main/NF.jsonld
+target_scope: https://github.com/cBioPortal/clinical-data-dictionary/blob/master/docs/resource_uri_to_clinical_attribute_mapping.txt
+mapping:
+  - source: individualID
+    target:
+      label: PATIENT_ID
+      description: Patient ID
+      attribute_type: PATIENT
+      data_type: STRING
+  - source: sex
+    target:
+      label: SEX
+      description: Sex of the patient
+      attribute_type: PATIENT
+      data_type: STRING
+  - source: nf1Genotype
+    target:
+      label: NF1_GENOTYPE
```

Comment:
```
C002173	NF1_GERMLINE
C002174	NF1_MUTATION
C002175	NF1_SOMATIC
```

Reply:
I'll be meeting with MSKCC people to confirm that we can add this as well as a few other things.